python/lsst/faro/base/MatchedCatalogBase.py

```diff
@@ -156,6 +157,7 @@ class MatchedBaseConfig(
         default=False,
     )
 
+    _loggingInterval = 600
```

Comment:
Given that we briefly introduced `loggingInterval` as a config field in other packages before deprecating it, I feel like it'd be easier to hard-code the value here for v23. Depending on whether I should (or not) leave this as a deprecated config field, I will fixup or drop this commit.

Reply:
Private constant is fine with me. This is only v23 branch anyhow.